web/includes/functions.php

```diff
@@ -2129,4 +2129,37 @@ function getStreamHTML( $monitor, $scale=100 ) {
     }
 } // end function getStreamHTML
 
+function check_timezone() {
+  $system_timezone = '';
+  if (is_link('/etc/localtime')) {
```

Comment:
problem is that the contents of the file are not really parsable into a timezone.   The fact that it is a link is useful because we can read the timezone from the actual file name.